docs/platforms.md

```diff
@@ -50,6 +50,17 @@
 | BIRD Internet Routing Daemon [❗](caveats-bird) | bird               |
 | dnsmasq DHCP server [❗](caveats-dnsmasq)       | dnsmasq            |
 
+(platform-host)=
+Most devices behave as routers (or layer-3 switches); the following devices can take multiple roles or behave as [IP hosts](node-router-host):
+
+| Device | router | host |
+|-----------------------|:--:|:--:|
+| Arista EOS            | ✅ | ✅ |
```

Comment:
Would be good to add FRR to this list

Reply:
Currently it's failing the `initial/05-host` test for several reasons. We have to modify the configuration templates to make it work as a proper host.